notebooks/02_model/als_deep_dive.ipynb

```diff
@@ -87,16 +87,16 @@
   },
```

Comment:
<p>the change in the rank list is the only change right? how much faster is the computation now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/864/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
It was run on my DSVM which has a different size - on mine the run time can be reduced to approx. 1/3 of the original.